examples/stocks/test/locale_test.dart

```diff
@@ -13,10 +13,13 @@ void main() {
     stocks.main();
     await tester.idle(); // see https://github.com/flutter/flutter/issues/1865
     await tester.pump();
+    // Note that the initial test app's locale is "_", so we're seeing
+    // the fallback translation here.
     expect(find.text('MARKET'), findsOneWidget);
     await tester.binding.setLocale('es', 'US');
     await tester.idle();
     await tester.pump();
+    await tester.pumpAndSettle(); // Wait for the localized resources to load.
```

Comment:
It now takes two frames: one for the LocalizedResources widget to get built with the new target locale and one for async load to finish and trigger another build.